.circleci/config.yml

```diff
@@ -1,15 +1,15 @@
 version: 2.1
 
 orbs:
-  android: circleci/android@2.0
+  android: circleci/android@2.4.0
   codecov: codecov/codecov@3.2.4
 
 jobs:
   build-and-unit-test:
     executor:
       name: android/android-machine
       resource-class: large
-      tag: 2022.01.1
+      tag: 2024.01.1
```

Comment:
Since our unit tests do no require an emulator instance can we switch to using a android docker image. Same comment for all our unit test jobs.
```
executor:
      name: android/android-docker
      tag:...
```